src/Views/gagepage.html

```diff
@@ -240,20 +240,21 @@ <h4 class="modal-title">StreamStats Gage Page</h4>
                         <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.heatChartConfig">
                             <highchart id="chart2" config="vm.heatChartConfig"></highchart>
                         </div>
-                    </center> -->
+                    </center>
                     <center>
-                        <div ng-if="!vm.dischargeChartConfig" class="panel panel-default reportSpinner">
+                        <div ng-if="!vm.chartConfig" class="panel panel-default reportSpinner">
                             &nbsp;<i class="fa fa-spinner fa-3x fa-spin loadingSpinner"></i>
                         </div>
-                        <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.dischargeChartConfig">
-                        <input type="checkbox" id="logScaleDischarge" name="logScaleDischarge" ng-model="vm.logScaleDischarge" ng-change="vm.toggleLogLinearDischarge()">
-                        <label for="logScaleDischarge" > Log Scale</label>
+                        <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.chartConfig">
+
+                        <input type="checkbox" ng-if="vm.formattedDischargePeakDates.length > 0" id="logScaleDischarge" name="logScaleDischarge" ng-model="vm.logScaleDischarge" ng-change="vm.toggleLogLinearDischarge()" ng-hide="!data">
+                        <label for="logScaleDischarge" ng-if="vm.formattedDischargePeakDates.length > 0"> Log Scale</label>
                         
-                        <input type="radio" id="ageData" name="ageQualityData" ng-model="vm.ageQualityData" value="age" ng-change="vm.toggleDischargeData('age')"> 
-                        <label for="ageData"> Age</label>
+                        <input type="radio"  ng-if="vm.measuredObj.control !== undefined" id="ageData" name="ageQualityData" ng-model="vm.ageQualityData" value="age" ng-change="vm.toggleDischargeData('age')" checked> 
```

Comment:
I don't think the ng-if is working as expected. For example, in this graph for gage 12340000 I am seeing that USGS measured data is available, but the Age/Quality radio buttons aren't visible. This is because this.measuredObj.control is undefined. 

![image](https://user-images.githubusercontent.com/40237491/228266034-a0a7c97f-70a6-40e4-acdf-c820a8c8ffed.png)

Should it be `ng-if="vm.measuredObj !== undefined"` ?